### PR TITLE
Add guide on embedding an ExperienceFragment in an Activity

### PR DESCRIPTION
**Pull request by gabrielbunselmeyer**
Closes https://github.com/judoapp/judo-developer-site/issues/12.

### CODE REVIEW

File: _android/embedding.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    android:layout_height="match_parent" />
+```
+
+Judo comes with a handy method for creating intents based on a given experience url: `Judo.makeIntent`. You can use this to easily populate intents with the relevant experience data inside your `AppCompatActivity`, for example:
+
+```kotlin
+val intent = Judo.makeIntent(
+    context = this,
+    url = url,
+    ignoreCache = true
```
Comment (chrisrecalis) [inline _android/embedding.md:40]: Can we exclude this param from doc or use false as we don’t want to lead customers copying this and bypassing cache. Same goes for the sample code below

File: _android/embedding.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    ...
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        ...
+
+        launchButton.setOnClickListener {
+            val intent = Judo.makeIntent(
+                context = this,
+                url = url,
+                ignoreCache = true
```
Comment (chrisrecalis) [inline _android/embedding.md:117]: Same here

File: _android/embedding.md (hunk 1 above)
Comment (gabrielbunselmeyer) [reply _android/embedding.md:40]: Good point!

File: _android/embedding.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+With that said, correctly sizing up your `Fragment` container should work in most cases.
+
+Inside our `Activity`'s XML:
+
+```xml
+<androidx.fragment.app.FragmentContainerView
+    android:id="@+id/fragment_container_view"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent" />
```
Comment (orospakr) [inline _android/embedding.md:31]: this example contradicts the previous statement - but in lieu of putting an arbitrary example size here, how about put a comment like Biso did on the iOS version stating that a "fixed size is required here"?

**Review by orospakr — APPROVED** [review]

**Pull request merged**